ethminer/MinerAux.h

```diff
@@ -150,35 +145,6 @@ class MinerCLI
 				cerr << "Bad " << arg << " option: " << argv[i] << endl;
 				BOOST_THROW_EXCEPTION(BadArgument());
 			}
-#if ETH_ETHASHCL
-		else if (arg == "--cl-global-work" && i + 1 < argc)
-			try {
-				m_globalWorkSizeMultiplier = stol(argv[++i]);
-			}
-			catch (...)
-			{
-				cerr << "Bad " << arg << " option: " << argv[i] << endl;
-				BOOST_THROW_EXCEPTION(BadArgument());
-			}
-		else if (arg == "--cl-local-work" && i + 1 < argc)
-			try {
-				m_localWorkSize = stol(argv[++i]);
-			}
-			catch (...)
-			{
-				cerr << "Bad " << arg << " option: " << argv[i] << endl;
-				BOOST_THROW_EXCEPTION(BadArgument());
-			}
-		else if (arg == "--cl-ms-per-batch" && i + 1 < argc)
-			try {
-				m_msPerBatch = stol(argv[++i]);
-			}
-			catch (...)
-			{
-				cerr << "Bad " << arg << " option: " << argv[i] << endl;
-				BOOST_THROW_EXCEPTION(BadArgument());
-			}
-#endif // ETH_ETHASHCL
 		else if (arg == "--list-devices")
```

Comment:
I think if we are concerned about compatibility, we should be more concerned about compatibility with geth than with older versions of eth.